src/test/regress/multi_schedule

```diff
@@ -74,6 +74,7 @@ test: custom_aggregate_support
 test: multi_average_expression multi_working_columns multi_having_pushdown
 test: multi_array_agg multi_limit_clause multi_orderby_limit_pushdown
 test: multi_jsonb_agg multi_jsonb_object_agg multi_json_agg multi_json_object_agg bool_agg
+test: ch_bench_having
```

Comment:
can it run in parallel with another test? if not, can we add a comment on why this should remain separate?